App.tsx

```diff
@@ -15,13 +16,28 @@ const setInitalTime = () => {
 };
 
 const App = () => {
+  const [fontLoaded, setFontLoaded] = React.useState(false);
+  const [reminderEnabled, setReminderEnabled] = React.useState(true);
   const [screen, setScreen] = React.useState<ScreensT>("home");
   const [frequency, setFrequency] = React.useState(60);
   const [startTime, setStartTime] = React.useState(setInitalTime());
   const [endTime, setEndTime] = React.useState(
     dateFns.addHours(setInitalTime(), 12)
   );
-
+  React.useEffect(() => {
```

Comment:
You could move this to a separate file like "theme.ts", and use a custom hook to load the font

Reply:
<img width="625" alt="Screen Shot 2020-03-26 at 12 12 39 PM" src="https://user-images.githubusercontent.com/13867964/77640690-231c8480-6f5b-11ea-945e-046a249e680b.png">

Reply:
like this?

Reply:
Close, see https://github.com/ErlendS/quotie/pull/3#discussion_r398537746